modules/light-clients/07-tendermint/client_state.go

```diff
@@ -246,6 +247,51 @@ func (cs ClientState) VerifyMembership(
 	return merkleProof.VerifyMembership(cs.ProofSpecs, consensusState.GetRoot(), merklePath, value)
 }
 
+// VerifyMultihopMembership is a generic proof verification method which verifies a multi-hop proof of the existence of a value at a given CommitmentPath at the specified height.
+// The caller is expected to construct the full CommitmentPath from a CommitmentPrefix and a standardized path (as defined in ICS 24).
+// If a zero proof height is passed in, it will fail to retrieve the associated consensus state.
+func (cs ClientState) VerifyMultihopMembership(
```

Comment:
This implementation actually should work for single-hop cases converted to multi-hop as well.